src/fstab-generator/fstab-generator.c

```diff
@@ -721,7 +721,7 @@ static int add_sysroot_mount(void) {
         else
                 opts = arg_root_options;
 
-        log_debug("Found entry what=%s where=/sysroot type=%s", what, strna(arg_root_fstype));
+        log_debug("Found entry what=%s where=/sysroot type=%s opts=%s", what, strna(arg_root_fstype), strna(opts));
```

Comment:
I think this should be `strempty(opts)`. After all, it's fine for opts to be empty.